### PR TITLE
Update pages (HTML, JS)

### PR DESCRIPTION
**Pull request by PedroRossa98**
This branch was all the changes needed to make the work full of the pages for the pendulum and for the dev tool MC.

### CODE REVIEW

**Comment by PedroRossa98** [comment]
If this is working correctly I will clean the console.log's and comment it better

**Review by airguru — COMMENTED** [review]
It doesn't work without this?? Any ideas how to make it automatic? Maybe if we add a mandatory prefix to the experiment directories?

File: freeweb/settings.py [hunk 1]
```diff
@@ -110,6 +110,11 @@
     SESSION_COOKIE_SECURE = True
     CSRF_COOKIE_SECURE = True
 
+LOCALE_PATHS = (
```
Comment (airguru) [inline freeweb/settings.py:113]: It doesn't work without this?? Any ideas how to make it automatic? Maybe if we add a mandatory prefix to the experiment directories?

Comment (PedroRossa98) [reply freeweb/settings.py:113]: We could do that. Like "page_"+...

Comment (bgeneto) [reply freeweb/settings.py:113]: > We could do that. Like "page_"+...

No need... just add a `PROJECT_DIR` like this (e.g. below `BASE_DIR` variable)

```
PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
````
to the `settings.py` file and then set our `LOCALE_PATHS` accordingly: 
```suggestion
LOCALE_PATHS = (
    os.path.join(PROJECT_DIR, 'locale'),
)
```

**Pull request merged**